index.js

```diff
@@ -201,6 +204,13 @@ module.exports = {
     self.upgradeDocIdentity = async doc => {
       const workflow = self.apos.modules['apostrophe-workflow'];
       doc.a2Id = doc._id;
+
+      // TODO: check if multisite?
```

Comment:
Missing the check.

Reply:
Check for `apos.options.multisite` which is only present in a multisite project.